examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
<p>Why can't we just use the draw() function on the PulseSequence as we did in the Native gates notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>